beacon_node/client/src/builder.rs

```diff
@@ -530,11 +509,10 @@ where
             }
             ClientGenesis::DepositContract => {
                 info!(
-                    context.log(),
-                    "Waiting for eth2 genesis from eth1";
-                    "eth1_endpoints" => format!("{:?}", &config.eth1.endpoint),
-                    "contract_deploy_block" => config.eth1.deposit_contract_deploy_block,
-                    "deposit_contract" => &config.eth1.deposit_contract_address
+                    eth1_endpoints = format!("{:?}", &config.eth1.endpoint),
```

Comment:
I think you should be able to do `eth1_endpoints = ?config.eth1.endpoint` instead of using `format!`